app/lib/frontend/templates/views/pkg/score_tab.dart

```diff
@@ -179,18 +180,35 @@ d.Node _section(ReportSection section) {
 }
 
 d.Node _downloadsChart(WeeklyVersionDownloadCounts weeklyVersionDownloads) {
+  final versionModes = d.div(
+    classes: ['downloads-chart-version-modes'],
+    children: [
+      radioButtons(
+          leadingText: 'By versions: ',
+          name: 'version-modes',
+          idsAndValues: [
+            (id: 'major', value: 'Major'),
+            (id: 'minor', value: 'Minor'),
+            (id: 'patch', value: 'Patch')
```

Comment:
```suggestion
            (id: 'version-modes-major', value: 'major', label: 'Major'),
            (id: 'version-modes-minor', value: 'minor', label: 'Minor'),
            (id: 'version-modes-patch', value: 'patch', label: 'Patch')
```

To avoid conflicting ids

Reply:
done